app/src/main/java/com/benoitquenaudon/daggerbundleinjectiondemo/ui/BaseAppCompatActivity.java

```diff
@@ -4,21 +4,26 @@
 import android.support.annotation.Nullable;
 import com.benoitquenaudon.daggerbundleinjectiondemo.BundleService;
 import dagger.android.support.DaggerAppCompatActivity;
+import javax.inject.Inject;
 
 public abstract class BaseAppCompatActivity extends DaggerAppCompatActivity {
-  private BundleService bundleService;
+  @Inject BundleService bundleService;
+  private Bundle savedInstanceState;
```

Comment:
Since I noticed the way the BundleModule was providing the service through the BaseActivity. I figured just have it wrap all it's state/bundles and have the service put them on injection.